frontend/src/FullPageRoutes/ForgotPassword.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react"
+import {
+  Grid,
+  makeStyles,
+  Button,
+  Typography,
+  TextField,
+} from "@material-ui/core"
+import { useState } from "react"
+import Notification from "../components/Notification"
+import { NotificationType } from "../components/Notification"
+import { parseIdentity, request } from "../utils/fetch"
+
+export type UserAccounts = {
+  first_name: string
+  last_name: string
+  username: string
+  password: string
+  email: string
+  successor: string
```

Comment:
For the sake of my getting a better understanding of what's going on, why are we pulling out first name, last name and successor?